src/components/calcite-combobox/calcite-combobox.tsx

```diff
@@ -299,57 +402,56 @@ export class CalciteCombobox {
   //--------------------------------------------------------------------------
 
   render(): VNode {
-    const dir = getElementDir(this.el);
+    const { active, disabled, el, label, placeholder, scale, selectedItems } = this;
+    const dir = getElementDir(el);
     const listBoxId = "listbox";
     return (
-      <Host
-        active={this.active}
-        onFocusin={this.comboboxFocusHandler}
-        onFocusout={this.comboboxFocusHandler}
-        dir={dir}
-      >
+      <Host active={active} dir={dir}>
         <div class="selections">
-          {this.selectedItems.map((item) => {
+          {selectedItems.map((item) => {
             return (
-              <calcite-chip
-                key={item.value}
-                scale={this.scale}
-                value={item.value}
-                dir={dir}
-                dismissible
-              >
+              <calcite-chip key={item.value} scale={scale} value={item.value} dir={dir} dismissible>
                 {item.textLabel}
               </calcite-chip>
             );
           })}
         </div>
         <div
           role="combobox"
-          aria-expanded={this.active.toString()}
+          aria-expanded={active.toString()}
           aria-owns={listBoxId}
           aria-haspopup="listbox"
+          ref={this.setReferenceEl}
         >
           <input
+            onFocus={this.comboboxFocusHandler}
```

Comment:
Nitpick: For readability, can you sort the attributes? Makes it easier to scan. I'll see if there's a rule we can enable to fix this automatically.

Reply:
+1 for alpha sorting!  #alphasortallthethings

Reply:
You can ignore this one. I've got a PR to auto sort hot off the press: https://github.com/Esri/calcite-components/pull/978

Reply:
w00t